index.bs

```diff
@@ -21,6 +21,7 @@ Repository: w3c/sensors
 Markup Shorthands: markdown on
 Boilerplate: omit issues-index, omit conformance, omit feedback-header
 Default Biblio Status: current
+Inline Github Issues: true
```

Comment:
Well, it previously was related to this PR, as I first introduced an inline GH issue here. I've added other issues since, so that will go when we rebase.